src/client/components/Diagram.tsx

```diff
@@ -23,32 +164,252 @@ class Diagram extends React.Component<DiagramProps> {
             height: this.divRef.current.getBoundingClientRect().height,
             gridSize: 1,
         });
+        paper.setInteractivity({ elementMove: false });
 
-        const rect = new joint.shapes.standard.Rectangle();
-        rect.position(100, 30);
-        rect.resize(100, 40);
-        rect.attr({
-            body: {
-                fill: 'blue',
-            },
-            label: {
-                text: 'Hello',
-                fill: 'white',
-            },
-        });
-        rect.addTo(graph);
+        const myObjStr = JSON.stringify(MOCK_DATA);
+        const data = JSON.parse(myObjStr);
```

Comment:
You should just be able to use `MOCK_DATA` directly

Reply:
When I use MOCK_DATA directly I get warnings when I start parsing the body of data. E.g. 
`const {processes} = MOCK_DATA`, and then when I start to generate the process tables it will mention a type error for using `processes`...

Reply:
This works without lint errors for me:
```
const { processes, openFiles, vnodes } = MOCK_DATA;
```
It's possible the errors were previously caused by lack of type definitions for `MOCK_DATA`.